src/nanorc/core.py

```diff
@@ -3,6 +3,7 @@
 import copy as cp
 import json
 import os
+import copy as cp
```

Comment:
Needed or leftover?

Reply:
Most of these import are useless indeed.